src/main/resources/schemas/schema-form.json

```diff
@@ -34,6 +34,12 @@
       "default": 2000,
       "minimum": 0
     },
+    "releaseCache": {
+      "title": "Release the cache when on stop the API ?",
+      "description": "If you enable, the resource won't release the cache. To do so, you have to manage it yourself on your Redis server",
```

Comment:
`If you enable, the resource won't release the cache. To do so, you have to manage it yourself on your Redis server` => `If enabled, the resource will release the cache. If not, you will have to manage it by yourself on your Redis server.`